dataplicity/m2m/proxy.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, user=None, group=None, size=None):
             size = [80, 24]
         self.size = size
         self.master_fd = None
+        self.pid = None
```

Comment:
Fixes a race condition when a channel is closed before the remote process has spawned. Not sure if this bug had any impact on the end-user. But it did result in a traceback in the logs.